docs/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+[EVG-<number>](https://jira.mongodb.org/browse/EVG-<number>)
+
+### Description 
+< add description, context, thought process, etc >
+
+### Screenshots
+ < add screenshots of visible changes >
+
+### Testing 
+  < add a description of how you tested it >
+
+### Evergreen PR 
+  < link to a corresponding evergreen PR if applicable >
```

Comment:
Since we don't use `docs/` for anything else, how about adding this file to the `.github/` directory (which is another [valid path](pull_request_template.md))?